server/pkg/utils/email/email.go

```diff
@@ -18,6 +19,69 @@ import (
 	"github.com/spf13/viper"
 )
 
+func SendSES(toEmails []string, fromName string, fromEmail string, subject string, htmlBody string, inlineImages []map[string]interface{}) error {
```

Comment:
Since this is a private method, the convention is to start with a smaller case. Also, we could call it `sendViaSMTP` (again discarding the reference to SES)